src/ray/object_manager/plasma/store.cc

```diff
@@ -51,11 +51,19 @@
 #include "ray/object_manager/plasma/malloc.h"
 #include "ray/object_manager/plasma/plasma_allocator.h"
 #include "ray/object_manager/plasma/protocol.h"
+#include "ray/stats/stats.h"
 #include "ray/util/util.h"
 
 namespace ph = boost::placeholders;
 namespace fb = plasma::flatbuf;
 
+DEFINE_stats(num_pending_requests,
+             "The number of pending object creation requests in the queue.", (), (),
+             ray::stats::GAUGE);
+DEFINE_stats(num_pending_bytes,
```

Comment:
Could we put all the stats defs in a common file? Then we can put an OWNERS file on it to enforce stats standards.

Reply:
This required a bit of refactoring changes, but I think it is working now.